cmd/create.go

```diff
@@ -103,15 +103,16 @@ func createGenesis(cmd *cobra.Command, args []string) error {
 		}
 
 		var genesisBytes []byte
+		var tokenName string
 
 		switch subnetType {
 		case subnetEvm:
-			genesisBytes, err = vm.CreateEvmGenesis(subnetName, log)
+			genesisBytes, tokenName, err = vm.CreateEvmGenesis(subnetName, log)
 			if err != nil {
 				return err
 			}
 
-			err = createSidecar(subnetName, models.SubnetEvm)
+			err = createSidecar(subnetName, models.SubnetEvm, tokenName)
```

Comment:
Reminder for the 
```
if err := ....; err != nil {
```
pattern for 1-value assignments from funcs

Reply:
good catch, addressed in a few places